app/controllers/albums.js

```diff
@@ -0,0 +1,10 @@
+const albumsService = require('../services/albums');
+
+exports.getAlbums = (req, res, next) =>
+  albumsService
+    .albums()
```

Comment:
Change this name for something more descriptive like `getAlbums` or something like that